test/cli/samples/fail-after-warnings/_expected.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+require('unknown');
```

Comment:
don't think this is been checked with `error: () => true` in this or other tests. will investigate

Reply:
Please do. I checked e.g. `node-config-not-found` and changing the expected output there turned the test red.

Reply:
In this case it's because I wasn't returning `true` to make it continue. In the other tests where _expected.js was being ignored I removed `_expected.js` because they didn't match, and I'm not sure if the intent was to check the output or not?

Reply:
You need to return `true` in order to continue https://github.com/rollup/rollup/blob/8333387aeb2e839d720b86921878a8afd73bde4a/test/cli/index.js#L42
`_expected.js` should be compared to stdout, but only if there is no `_expected` dir, `config.test`, `config.result`, `config.execute`, `config.stderr` that does not return `true` or `config.error` that does not return `true`.